.circleci/config.yml

```diff
@@ -0,0 +1,15 @@
+version: 2.1
+orbs:
+  ruby: circleci/ruby@0.1.2 
+
+jobs:
+  build:
+    docker:
+      - image: circleci/ruby:2.6.3-stretch-node
```

Comment:
I know this is slightly different than our current one. Digging into the docs to see what our options are.